libgadget/tests/test_timebinmgr.c

```diff
@@ -64,27 +62,27 @@ static void test_conversions(void ** state) {
 
 static void test_skip_first(void ** state) {
 
-    setup_sync_points(All.TimeIC, All.TimeIC);
+    setup_sync_points(All.TimeIC, All.TimeMax, All.TimeIC, 0);
```

Comment:
Situations like this makes me wish we can use named arguments in C.

Reply:
+100.

Even more annoying when, eg, arguments of type double may be silently converted to int...